integration_tests/suite/test_meetings.py

```diff
@@ -288,7 +288,7 @@ def test_participant_joins_sends_event(self):
                 name='meeting',
             ),
         )
-        bus_events = self.bus.accumulator(f'meetings.{meeting_uuid}.participants.joined')
+        bus_events = self.bus.accumulator(headers={'meeting_uuid': meeting_uuid})
```

Comment:
I don't remember why we chose to use `user_uuid:<UUID>: true`. Should we use the same pattern for meetings?